src/tcgwars/logic/impl/gen4/LegendsAwakened.groovy

```diff
@@ -3195,14 +3211,22 @@ public enum LegendsAwakened implements LogicCardInfo {
           pokePower "Trade Off", {
             text "Once during your turn , you may look at the top 2 cards of your deck, choose 1 of them, and put it into your hand. Put the other card on the bottom of your deck. This power can’t be used if Uxie is affected by a Special Condition. You can’t use more than 1 Trade Off Poké-Power each turn."
             actionA {
+              checkNoSPC()
+              checkLastTurn()
+              assert my.deck : "Your deck is empty"
+              powerUsed()
+              def sel = my.deck.subList(0,2).select("Choose 1 card to put into your hand")
+              my.deck.subList(0,2).getExcludedList(sel).moveTo(suppressLog: true, my.deck)
+              sel.moveTo(hidden: true, my.hand)          
             }
           }
           move "Zen Blade", {
             text "60 damage. Uxie can’t use Zen Blade during your next turn."
             energyCost C, C
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 60
+              cantAttackNextTurn self
```

Comment:
According to the text, Uxie can still attack, it just can't use this same attack.
```suggestion
              cantUseAttack thisMove, self
```